pages/Magical-Armor.md

```diff
@@ -0,0 +1,28 @@
+The Magical Armor category contains various armour sets and pieces each with their own ability.  
+Below you will find info on all of them.
+
+## Magical Armor Sets
+### The Ender Set
+When wearing the Ender Boots, you stop taking damage from enderpearls.
+
+### The Leather & Iron Slime Set
+When wearing the Slime Boots, you gain Jump Boost IV and you negate fall damage.  
+When wearing the Slime Leggings, you gain Speed III.
+
+### The Glowstone Set
+When wearing any piece of the Glowstone Set, you gain Night Vision (I).
+
+### The Soulbound Set
+Any piece of this set will be kept on your death. You'll never lose it!
+
+### The Bee Set
+When wearing the Bee Wings (elytra), you will gain Slow Falling (I) when you get near the ground.  
+When wearing the Bee Boots, you gain Jump Boost III and you negate fall damage.
```

Comment:
I think it would be better to call them "Ender Armor", "Slime Armor", "Glowstone Armor", ....
respectively.